rubyUtils.gradle

```diff
@@ -168,8 +168,8 @@ void setupJruby(File projectDir, File buildDir) {
 Object executeJruby(File projectDir, File buildDir, Closure<?> /* Object*/ block) {
     def jruby = new ScriptingContainer()
     def env = jruby.environment
-    def gemDir = "${projectDir}/vendor/bundle/jruby/2.6.0".toString()
-    jruby.setLoadPaths(["${projectDir}/vendor/bundle/jruby/2.6.0/gems/bundler-2.4.13/lib".toString(), "${projectDir}/vendor/jruby/lib/ruby/stdlib".toString()])
+    def gemDir = "${projectDir}/vendor/bundle/jruby/3.1.0".toString()
+    jruby.setLoadPaths(["${projectDir}/vendor/bundle/jruby/3.1.0/gems/bundler-2.4.13/lib".toString(), "${projectDir}/vendor/jruby/lib/ruby/stdlib".toString()])
```

Comment:
either we apply the same change as in https://github.com/elastic/logstash/pull/15066, or need to update the path to be bundler-2.4.14, released 3 days ago https://rubygems.org/gems/bundler/versions/2.4.14